packages/scripts/src/addOrbitChain/schemas.ts

```diff
@@ -5,6 +5,62 @@ import { getOctokit } from "@actions/github";
 export const TESTNET_PARENT_CHAIN_IDS = [11155111, 421614, 17000, 84532];
 const ZERO_ADDRESS = constants.AddressZero;
 
+export const getParentChainInfo = (parentChainId: number) => {
+  switch (parentChainId) {
+    case 1: // Ethereum Mainnet
+      return {
+        rpcUrl: "https://eth.llamarpc.com",
+        blockExplorer: "https://etherscan.io",
+        chainId: 1,
+        name: "Ethereum",
+      };
+    case 42161: // Arbitrum One
+      return {
+        rpcUrl: "https://arb1.arbitrum.io/rpc",
+        blockExplorer: "https://arbiscan.io",
+        chainId: 42161,
+        name: "Arbitrum One",
+      };
+    case 11155111: // Sepolia
+      return {
+        rpcUrl: "https://ethereum-sepolia-rpc.publicnode.com",
+        blockExplorer: "https://sepolia.etherscan.io",
+        chainId: 11155111,
+        name: "Sepolia",
+      };
+    case 421614: // Arbitrum Sepolia
+      return {
+        rpcUrl: "https://sepolia-rollup.arbitrum.io/rpc",
+        blockExplorer: "https://sepolia.arbiscan.io",
+        chainId: 421614,
+        name: "Arbitrum Sepolia",
+      };
+    case 17000: // Holesky
+      return {
+        rpcUrl: "https://ethereum-holesky-rpc.publicnode.com	",
```

Comment:
we've got empty space here, not sure if it's affecting anything but good to remove